internal/cephfs/volumemounter.go

```diff
@@ -154,12 +154,13 @@ func mountFuse(ctx context.Context, mountPoint string, cr *util.Credentials, vol
 		"-c", util.CephConfigPath,
 		"-n", cephEntityClientPrefix + cr.ID, "--keyfile=" + cr.KeyFile,
 		"-r", volOptions.RootPath,
-		"-o", "nonempty",
 	}
 
+	fmo := "nonempty"
 	if volOptions.FuseMountOptions != "" {
-		args = append(args, ","+volOptions.FuseMountOptions)
+		fmo += "," + strings.TrimSpace(volOptions.FuseMountOptions)
```

Comment:
@nixpanic is it possible to cover an E2E for this one?

Reply:
Sure, we could create a StorageClass with the `debug` parameter and mount the PVC.

Reply:
e2e tests with ceph-fuse have been added, lets see if those work :crossed_fingers: